recipes/cgen/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cgen" %}
+{% set version = "2016.2" %}
+{% set sha256 = "879b7fe1842a1352591e202d1c33da2aa38b5dfe82ba59493b2b6bba6d5d7ce5" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - pytools
+        - six
+    run:
+        - python
+        - pytools
+        - numpy
+        - six
+
+test:
+    requires:
+        - pytest
+    imports:
+        - {{ name }}
+    commands:
+         - cd $SRC_DIR/test && python -m pytest  # [unix]
```

Comment:
I think `&&` behaves similarly on windows in the command prompt as it does in unix-land. If you want the tests run, you might see this might work generically on all platforms.